api/middleware.js

```diff
@@ -146,7 +164,7 @@ const parseJSON = function(field) {
       req[location][path] = JSON.parse(value);
       return true;
     } catch(e) {
-      throw new Error(format('could not parse field %s to a json', path));
+      throw new Error(format('Could not parse field %s to a json.', path));
```

Comment:
`to a json` is an odd construction. How about `to JSON`?
(I know this isn't from you)